src/measurement_kit/utils.cpp

```diff
@@ -16,8 +18,8 @@ BaseTest &common_init(std::list<Callback<BaseTest &>> il, BaseTest &test) {
             fflush(stdout);
         })
         .on_overall_data_usage([](DataUsage du) {
-            printf("Overall data usage (bytes): %llu down - %llu up\n",
-                    du.down, du.up);
+            printf("Overall data usage (bytes): %" PRId64 " down - %"
+                    PRId64 " up\n", du.down, du.up);
```

Comment:
This fixes a warning when building on Ubuntu on Windows. Not strictly related to the PR. If merging is not squashing, this would be included in master as a separate commit.